confbatstest/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM docker:19.03.2
+
+LABEL version="1.0.0"
+LABEL repository="http://github.com/redhat-cop/github-actions"
+LABEL homepage="http://github.com/redhat-cop/github-actions/confbatstest"
+LABEL maintainer="Red Hat CoP"
+LABEL "com.github.actions.name"="confbatstest"
+LABEL "com.github.actions.description"="Run conftest using BATS"
+LABEL "com.github.actions.icon"="package"
+LABEL "com.github.actions.color"="purple"
+
+RUN apk add --update bash git
+
+RUN export CONFTEST_VERSION=0.18.2 && \
+    wget https://github.com/instrumenta/conftest/releases/download/v${CONFTEST_VERSION}/conftest_${CONFTEST_VERSION}_Linux_x86_64.tar.gz && \
```

Comment:
Looks like this repo has now moved to the `open-policy-agent` org, https://github.com/open-policy-agent/conftest/releases. The old one redirects properly but feels appropriate to use the OPA repo here.